### ISSUE TITLE
Improve component name formatting

### ISSUE THREAD

**Issue by johnbradley** [Duke-GCB/bespin-ui#123]
## Details
CWL workflow fields with underscores do not display well:
<img width="430" alt="screen shot 2018-06-04 at 1 19 09 pm" src="https://user-images.githubusercontent.com/1024463/40931675-e5dad65e-67f9-11e8-8390-0f4e6c4a9e7e.png">

One possibility may be to replace underscores with spaces and capitalize each word.
One downside of this would be acronyms. Example name: `ancom_collapse_level` first word should be `ANCOM`.

## Code
The per-workflow input fields include a label based on the `name` value for each field in `answerSet.questionnaire.userFieldsJson`.
Code that reads json:
https://github.com/Duke-GCB/bespin-ui/blob/3c271ba5c7c4a087cfd368623383d62de2d623fd/app/components/questionnaire/answer-form-list.js#L8-L22

Example place that performs formatting of name:
https://github.com/Duke-GCB/bespin-ui/blob/d718b98513786a5fd8d3fe319058487439e88d10/app/components/questionnaire/answerable-field.js#L10-L17

**Comment by johnbradley** [comment]
For special cases like `ancom_collapse_level` we could read the `label` from the CWL workflow [InputParameter](https://www.commonwl.org/v1.0/Workflow.html#InputParameter). Otherwise `replace underscores with spaces and capitalize each word` in `name` as suggested above when there is no `label`.

**Comment by dleehr** [comment]
It makes sense to use the inputParameter's label in all cases - it seems like the intended use case, and we should be populating those in our workflows. But I think we'd need to capture this data when we import the workflow.

**Comment by johnbradley** [comment]
I really like the simplicity of using CWL `label` for our displayed label. How would this work with the currently loaded workflows. Manually fix the user parameters JSON in the database to have labels?

**Comment by johnbradley** [comment]
I did some testing and the workflow [loading code in bespin-api](https://github.com/Duke-GCB/bespin-api/blob/c18746d9a9cc657733f4cbc71348946a783cc2d9/data/importers.py#L173-L177) already loads all properties for a user input field:
```
        # Extract fields that are not system-provided
        user_fields = []
        for input_field in self.cwl_document.input_fields:
            if not input_field.get('name') in self.system_job_order_dict:
                user_fields.append(input_field)
```

**Comment by johnbradley** [comment]
Fixed by #133
